3.36s$ npm test
> socket.io@2.1.1 test /home/travis/build/socketio/socket.io
> nyc mocha --reporter spec --slow 200 --bail --timeout 10000 test/socket.io.js
  socket.io
    ✓ should be the same version as client
    set
      ✓ should be able to set ping timeout to engine.io
      ✓ should be able to set ping interval to engine.io
      ✓ should be able to set transports to engine.io
      ✓ should be able to set maxHttpBufferSize to engine.io
      ✓ should be able to set path with setting resource
      ✓ should be able to set origins to engine.io
      ✓ should be able to set authorization and send error packet
      ✓ should be able to set authorization and succeed
      ✓ should set the handshake BC object
    server attachment
      http.Server
        ✓ should serve static files
        ✓ should handle 304
        ✓ should not serve static files
        ✓ should work with #attach
      port
        ✓ should be bound
        1) should be bound as a string
        2) should be bound as a string
  15 passing (253ms)
  2 failing
  1) socket.io server attachment port should be bound as a string:
     Error: ECONNREFUSED: Connection refused
      at Test.assert (node_modules/supertest/lib/test.js:165:15)
      at localAssert (node_modules/supertest/lib/test.js:131:12)
      at node_modules/supertest/lib/test.js:128:5
      at Test.Request.callback (node_modules/superagent/lib/node/index.js:728:3)
      at ClientRequest.req.once.err (node_modules/superagent/lib/node/index.js:647:10)
      at Socket.socketErrorListener (_http_client.js:314:9)
      at emitErrorNT (net.js:1290:8)
      at _combinedTickCallback (internal/process/next_tick.js:80:11)
      at process._tickCallback (internal/process/next_tick.js:104:9)
  2) socket.io server attachment port should be bound as a string:
     Error: ECONNREFUSED: Connection refused
      at Test.assert (node_modules/supertest/lib/test.js:165:15)
      at localAssert (node_modules/supertest/lib/test.js:131:12)
      at node_modules/supertest/lib/test.js:128:5
      at Test.Request.callback (node_modules/superagent/lib/node/index.js:728:3)
      at ClientRequest.req.once.err (node_modules/superagent/lib/node/index.js:647:10)
      at Socket.socketErrorListener (_http_client.js:314:9)
      at emitErrorNT (net.js:1290:8)
      at _combinedTickCallback (internal/process/next_tick.js:80:11)
      at process._tickCallback (internal/process/next_tick.js:104:9)
---------------------|----------|----------|----------|----------|-------------------|
File                 |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
---------------------|----------|----------|----------|----------|-------------------|
All files            |    59.04 |    45.16 |    56.36 |    59.94 |                   |
 client.js           |    57.14 |    36.11 |    58.82 |    57.69 |... 69,270,271,272 |
 index.js            |    69.75 |    60.74 |    74.19 |    71.04 |... 90,511,512,513 |
 namespace.js        |    67.71 |       50 |    61.11 |    66.67 |... 81,282,294,295 |
 parent-namespace.js |    14.29 |      100 |        0 |    15.79 |... 31,32,33,34,35 |
 socket.js           |    48.04 |    20.51 |    48.65 |    49.74 |... 61,564,567,571 |
---------------------|----------|----------|----------|----------|-------------------|
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
cache.2
store build cache
0.00s
1.94snothing changed, not updating cache
Done. Your build exited with 1.